python/cudf/cudf/tests/test_scalar.py

```diff
@@ -253,6 +253,23 @@ def test_generic_null_scalar_construction_fails(value):
         cudf.Scalar(value)
 
 
+@pytest.mark.parametrize(
+    "value, dtype", [(1000, "uint8"), (2**30, "int16"), (-1, "uint16")]
+)
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
+def test_scalar_out_of_bounds_pyint_fails(value, dtype):
+    # Test that we align with NumPy on scalar creation behavior from
+    # Python integers.
+    np_ver = np.lib.NumpyVersion(np.__version__)
```

Comment:
You can use `packaging.version.parse` here if you don't want to dip into `np.lib`

Reply:
Changed also here now, thanks.  (And rebased.)